kafka/src/test/groovy/io/micronaut/configuration/kafka/KafkaConfigurationSpec.groovy

```diff
@@ -9,13 +9,20 @@ import io.micronaut.context.env.EnvironmentPropertySource
 import io.micronaut.context.env.MapPropertySource
 import io.micronaut.context.exceptions.NoSuchBeanException
 import org.apache.kafka.clients.consumer.Consumer
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.Producer
 import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.kafka.common.serialization.StringSerializer
 import spock.lang.AutoCleanup
 import spock.lang.Issue
 import spock.lang.Specification
 
+import java.nio.charset.StandardCharsets
+
 import static io.micronaut.context.env.PropertySource.PropertyConvention.ENVIRONMENT_VARIABLE
 import static org.apache.kafka.clients.consumer.ConsumerConfig.*
+import static org.apache.kafka.clients.producer.ProducerConfig.*
```

Comment:
Can we eliminate these wildcard imports? It makes it difficult to tell where each constant is coming from when reading the code.

In this particular case, since the keys are so similar between the classes, and some values are even duplicated, I might prefer to see static imports not used for them at all, and instead import `ConsumerConfig` and `ProducerConfig` and then prefix all of the constants in the code below, i.e., `ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG`.

Reply:
Fair enough. This has created quite some noise though. `ConsumerConfig` imports were being used everywhere.